specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/examples/2018-01-01/call_participant_removed.json

```diff
@@ -0,0 +1,50 @@
+{
+  "source": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
+  "subject": "thread/{thread-id}/senderCommunicationIdentifierRawId/{rawId}/recipientCommunicationIdentifierRawId/{rawId}",
+  "type": "Microsoft.Communication.CallParticipantRemoved",
+  "time": "2021-02-19T00:25:59.9436666Z",
+  "id": "02272459-badb-4e2e-b538-4cb8a2f71da6",
+  "data": {
+    "reason": {
```

Comment:
This field isn't in the model for AcsCallParticipantRemovedEventData, or its parent objects. Is that intentional? I see the comment about it as resolved, but I'm not sure what the resolution was.

If the intention is to not expose it I'd recommend removing it from the example.